db/migrations/01_create_peeps_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, message VARCHAR(140), time VARCHAR(60));
```

Comment:
Maybe consider having message  column as not null - Users shouldn't be able create a blank peep